SAR_Training/English/Hazards/Prepare_RTC_Stack_HyP3_v2.ipynb

```diff
@@ -64,11 +64,8 @@
     "\n",
     "import asf_notebook as asfn\n",
     "\n",
-    "try:\n",
-    "    from hyp3_sdk import Batch\n",
-    "except:\n",
-    "    !python -m pip install hyp3-sdk==0.5 --user\n",
-    "    from hyp3_sdk import Batch"
+    "!python -m pip install hyp3-sdk --upgrade\n",
+    "from hyp3_sdk import Batch"
```

Comment:
Since we just bumped to `v1.0.0`, I think you'll want to write it like:

```
!python -m pip install hyp3-sdk~=1.0 --upgrade
```
We do follow symmantic versioning so that will protect against breaking changes, but keep it up to date.

Reply:
Thanks, Joe. I just added your suggested change.